src/other/precompile.jl

```diff
@@ -0,0 +1,3219 @@
+        Base.precompile(Tuple{Core.kwftype(typeof(DataFrames.reduce)),NamedTuple{(:cols,),Tuple{Symbol}},typeof(reduce),typeof(vcat),NTuple{6,DataFrame}})
+        Base.precompile(Tuple{Core.kwftype(typeof(DataFrames.Type)),NamedTuple{(:g1, :x1, :g2, :x2),Tuple{Array{Int,1},UnitRange{Int},Array{Int,1},UnitRange{Int}}},Type{DataFrame}})
+        Base.precompile(Tuple{Core.kwftype(typeof(DataFrames.Type)),NamedTuple{(:Key1, :Key2, :Value),Tuple{PooledArrays.PooledArray{String,UInt8,1,Array{UInt8,1}},PooledArrays.PooledArray{Union{Missing, String},UInt8,1,Array{UInt8,1}},UnitRange{Int}}},Type{DataFrame}})
+        Base.precompile(Tuple{typeof(DataFrames.do_call),typeof(Statistics.std),Array{Int,1},Array{Int,1},Array{Int,1},GroupedDataFrame{DataFrame},Tuple{Array{Union{Missing, DataFrame},1}},Int})
+        Base.precompile(Tuple{typeof(DataFrames.do_call),typeof(maximum),Array{Int,1},Array{Int,1},Array{Int,1},GroupedDataFrame{DataFrame},Tuple{Array{DataFrame,1}},Int})
+        Base.precompile(Tuple{Core.kwftype(typeof(DataFrames.Type)),NamedTuple{(:Key1, :Key2, :Value),Tuple{Array{Union{Missing, String},1},Array{String,1},UnitRange{Int}}},Type{DataFrame}})
+        Base.precompile(Tuple{Core.kwftype(typeof(DataFrames.Type)),NamedTuple{(:id1, :id2, :x, :ID2, :x_1),Tuple{Array{Int,1},Array{Union{Missing, Int},1},Array{Union{Missing, Int},1},Array{Union{Missing, Int},1},Array{Union{Missing, Int},1}}},Type{DataFrame}})
+        Base.precompile(Tuple{typeof(DataFrames._combine_multicol),NamedTuple{(:p, :q),Tuple{Array{Float64,1},Array{Float64,1}}},Function,GroupedDataFrame{DataFrame},Nothing})
+        Base.precompile(Tuple{typeof(DataFrames._combine_rows_with_first!),NamedTuple{(:x1,),Tuple{Float64}},Tuple{Array{Union{Missing, Float64},1}},Int,Int,Function,GroupedDataFrame{DataFrame},Tuple{Array{Union{Missing, Int},1}},Tuple{Symbol},Val{false}})
+        isdefined(DataFrames, Symbol("#67#74")) && Base.precompile(Tuple{getfield(DataFrames, Symbol("#67#74")),Array{Any,1}})
+        Base.precompile(Tuple{Core.kwftype(typeof(DataFrames.Type)),NamedTuple{(:id, :fid, :id_1),Tuple{Array{Union{Missing, Int},1},Array{Int,1},Array{Union{Missing, Int},1}}},Type{DataFrame}})
+        Base.precompile(Tuple{typeof(view),DataFrame,UnitRange{UInt},Colon})
+        Base.precompile(Tuple{Core.kwftype(typeof(DataFrames.rightjoin)),NamedTuple{(:on,),Tuple{Array{Pair{String,String},1}}},typeof(rightjoin),DataFrame,DataFrame})
+        Base.precompile(Tuple{typeof(DataFrames._combine_with_first),NamedTuple{(:x1,),Tuple{Float64}},Function,GroupedDataFrame{DataFrame},Tuple{Array{Union{Missing, Int, Int8},1}},Val{false},Array{Int,1}})
+        Base.precompile(Tuple{typeof(push!),DataFrame,Tuple{Int,String,Int}})
+        Base.precompile(Tuple{typeof(DataFrames.genkeymap),GroupedDataFrame{DataFrame},Tuple{Array{Symbol,1},Array{Int,1}}})
+        Base.precompile(Tuple{typeof(findall),Function,DataFram
```

Comment:
```suggestion
```

Reply:
same here

Reply:
This one (line 1594) seems to be the real
```julia
         Base.precompile(Tuple{typeof(findall),Function,DataFrames.DataFrameColumns{DataFrame}})
```
right?